entropy/windows/Raaz/Entropy.hsc

```diff
@@ -30,18 +31,19 @@ type HCRYPTPROV = Ptr ()
 foreign import WINDOWS_CCONV unsafe "Wincrypt.h CryptGenRandom"
     c_CryptGenRandom :: HCRYPTPROV -> Word32 -> Ptr Word8 -> IO Bool
 
-foreign import WINDOWS_CCONV unsafe "Wincrypt.h CryptAcquireContext"
+foreign import WINDOWS_CCONV unsafe "Wincrypt.h CryptAcquireContextW"
     c_CryptAcquireContext :: Ptr HCRYPTPROV -> CWString -> CWString
                           -> Word32 -> Word32 -> IO Bool
 
 foreign import WINDOWS_CCONV unsafe "Wincrypt.h CryptReleaseContext"
     c_CryptReleaseContext :: HCRYPTPROV -> Word32 -> IO Bool
 
 -- | Cache the crypto context so we don't have to create it on each call.
-cryptoContext :: ForeignPtr (HCRYPTPROV)
+cryptoContext :: ForeignPtr HCRYPTPROV
```

Comment:
Don't seem to have dieharder but generating random numbers seems to work correctly

```
Tamar@Rage /c/U/T/D/P/g/raaz> raaz rand -- 10 | od -x
0000000 4a90 c4b7 1ac0 6524 0478
0000012
Tamar@Rage /c/U/T/D/P/g/raaz> raaz rand -- 10 | od -x
0000000 14ce f1ba a478 195a 70ad
0000012
Tamar@Rage /c/U/T/D/P/g/raaz> raaz rand -- 10 | od -x
0000000 f613 7bf3 4549 a039 1e5e
0000012
Tamar@Rage /c/U/T/D/P/g/raaz> raaz rand -- 10 | od -x
0000000 d005 46ad 0b9a d148 0ecc
0000012
Tamar@Rage /c/U/T/D/P/g/raaz> raaz rand -- 10 | od -x
0000000 8a21 70cc 10c8 6c49 3159
0000012
```